src/main/java/com/iexec/worker/tee/TeeAbstractService.java

```diff
@@ -0,0 +1,26 @@
+package com.iexec.worker.tee;
+
+import com.iexec.common.task.TaskDescription;
+import com.iexec.sms.api.TeeSessionGenerationResponse;
+
+import javax.annotation.Nonnull;
+import java.util.Collection;
+import java.util.List;
+
+public interface TeeAbstractService {
+    boolean isTeeEnabled();
+
+    List<String> buildPreComputeDockerEnv(
+            TaskDescription taskDescription,
+            @Nonnull TeeSessionGenerationResponse session);
+
+    List<String> buildComputeDockerEnv(
+            TaskDescription taskDescription,
+            @Nonnull TeeSessionGenerationResponse session);
+
+    List<String> buildPostComputeDockerEnv(
+            TaskDescription taskDescription,
+            @Nonnull TeeSessionGenerationResponse session);
+
+    Collection<String> getBindings();
```

Comment:
```suggestion
    Collection<String> getAdditionalBindings();
```
?

Reply:
That's better, right!
https://github.com/iExecBlockchainComputing/iexec-worker/pull/464/commits/2949f847387d47eb5eea70249081712eff129db1.

Reply:
Thanks :)